### PR TITLE
Create Remove nodes from cluster topic.

### PR DESCRIPTION
**Pull request by KoryKessel-Mirantis**

### CODE REVIEW

**Review by byDimasik — APPROVED** [review]

**Comment by byDimasik** [comment]
LGTM. Thank you!

File: content/docs/operations/remove-node-from-cluster [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+weight: 6
+---
+
+The method for removing nodes from an MKE cluster depends on whether the node is
+a control plane node or a worker node.
+
+## Remove a control plane node
+
+Refer to the k0s documentation, [Remove or replace a controller](https://docs.k0sproject.io/stable/remove_controller/)
+for informaation on how to remove a control plane node from an MKE 4 cluster.
```
Comment (MagdaDziadosz) [inline content/docs/operations/remove-node-from-cluster:12]:
```suggestion
for information on how to remove a control plane node from an MKE 4 cluster.
```

File: content/docs/operations/remove-node-from-cluster [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Remove a worker node
+
+The removal of a worker node from an MKE 4 cluster is a simple delete
+operation.
```
Comment (MagdaDziadosz) [inline content/docs/operations/remove-node-from-cluster:17]:
```suggestion
To remove a worker node from an MKE 4 cluster:
```

**Review by MagdaDziadosz — CHANGES_REQUESTED** [review]

File: content/docs/operations/remove-node-from-cluster (hunk 2 above)
Comment (KoryKessel-Mirantis) [reply content/docs/operations/remove-node-from-cluster:17]: Removing the line entirely and editing Step 1.

File: content/docs/operations/remove-node-from-cluster [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+Refer to the k0s documentation, [Remove or replace a controller](https://docs.k0sproject.io/stable/remove_controller/)
+for information on how to remove a control plane node from an MKE 4 cluster.
+
+## Remove a worker node
+
+The removal of a worker node from an MKE 4 cluster is a simple delete
+operation.
+
+1. Run the following  `kubectl` command:
```
Comment (KoryKessel-Mirantis) [inline content/docs/operations/remove-node-from-cluster:19]:
```suggestion
1. Delete the worker node from the MKE 4 cluster:
```

**Review by MagdaDziadosz — APPROVED** [review]

**Pull request merged**